packages/components/src/components/Log/Log.js

```diff
@@ -28,17 +30,147 @@ const itemSize = 15; // This should be kept in sync with the line-height in SCSS
 const defaultHeight = itemSize * 100 + itemSize / 2;
 
 export class LogContainer extends Component {
-  state = { loading: true };
+  constructor(props) {
+    super(props);
+    this.state = { loading: true };
+    this.logRef = React.createRef();
+    this.textRef = React.createRef();
+  }
 
   componentDidMount() {
     this.loadLog();
+    window.addEventListener("scroll", this.handleLogScroll, true);
```

Comment:
Ideally we would only register the scroll listener when one of the features relying on it is enabled, similar with the other code here.